backend/app/db/alembic_dev/versions/a03982887391_add_relationship.py

```
@@ -0,0 +1,30 @@
+"""add relationship
+
+Revision ID: a03982887391
+Revises: 1d88d95ae783
+Create Date: 2024-11-30 01:14:55.238114
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
```

Ruff (0.8.0)

`alembic.op` imported but unused

Remove unused import: `alembic.op`

(F401)